src/pytest_mock_resources/container/mongo.py

```diff
@@ -1,10 +1,13 @@
 import pytest
 
 from pytest_mock_resources.compat import pymongo
-from pytest_mock_resources.config import DockerContainerConfig, fallback
+from pytest_mock_resources.config import DockerContainerConfig, fallback, get_env_config
 from pytest_mock_resources.container.base import ContainerCheckFailed, get_container
 
 
+mongo_image = get_env_config("mongo", "image") or "mongo:3.6"
```

Comment:
any reason in particular that you're changing this here also? Mostly I might expect this to make testing the env config loading more challenging because it's some global attribute.

Today it's sort of intentionally lazy and only evaluated after falling back through the other means of configuration. but it *should* already be being picked up in these fixture/config classes.

Reply:
@oakhan3 mentioned that we should have consistency across the entire library in terms of referencing images.

Reply:
Reviewing the docs it looks like we can get away with no changes if we leverage the config classes in the cli module.

If we use dynamic env vars like stated here: https://pytest-mock-resources.readthedocs.io/en/latest/docker.html?highlight=config#environment-variables

Reply:
agree. I think if you revert the changes to `container/*.py`, then it looks approvable to me

Reply:
@DanCardin See the changes in `cli.py`

Reply:
The changes in cli.py are great. It's the changes in `container/mongo.py`, `container/postgres.py`, and `container/mysql.py` that I dont think we want.

It's not necessarily, but more importantly it changes the env-var lookup semantics from being performed lazily to being performed on module load.

Reply:
@DanCardin but we are loading the image from version from `container/mongo.py`, `container/postgres.py`, `container/mysql.py` :thinking:. If I undo those changes, we won't be able to change the image when running `pmr`.

Reply:
All these config objects make use of the predefined `@fallback` decorator https://github.com/schireson/pytest-mock-resources/blob/master/src/pytest_mock_resources/config.py#L33, which means these `_field_defaults` values will only be the last option if no others are set.

If you are using `MongoConfig.image` in the `pmr` cli, then it will use the same fallback mechanism and make use of i.e. `PMR_MONGO_IMAGE`, if set, before falling back to the string literal version that was used originally.

Reply:
I see :thinking:. @DanCardin  I've reverted the changes :+1: